zipkin-lens/src/components/TracePage/TraceTimeline/TraceTree.jsx

```diff
@@ -294,6 +294,13 @@ const TraceTree = React.memo(
                 className={classes.serviceBadgeText}
               >
                 {serviceName}
+                <title>
```

Comment:
so this is all wrapped in `React.memo(` I suspect it is harder to test, does anyone know if we have an example that tests something like this?

we can merge, just so far one person said they cannot see tool tip as a result at the moment.. cc @openzipkin/ui

Reply:
Tooltips fine here on latest FF Ubuntu

Reply:
@drolando mind testing this for impact ^^ I think we don't have a massive test trace